### PR TITLE
minor improvements to chapter "Comparison with Bitcoin" part 3

### PR DESCRIPTION
**Pull request by 8go**
- commas
- more details
- etc.

### CODE REVIEW

File: ch03.asciidoc [hunk 1]
```diff
@@ -524,24 +524,37 @@ As payments are onion-routed, the users who route the payment will only be aware
...
 On the Bitcoin network, transactions are only 'settled' once they have been included in a block, in which case they are 'confirmed' in that block.
-As more blocks are mined on the Bitcoin network after this, the transaction acquires more 'confirmations' and is considered more secure.
+As more blocks are mined on the Bitcoin network thereafter, the transaction acquires more 'confirmations' and is considered more secure.
 
-On the Lightning Network, confirmation only matters for opening and closing channels.
+On the Lightning Network, confirmations only matters for opening and closing channels on-chain.
 Once a Funding Transaction has reached a suitable number of confirmations (e.g. 3), the channel partners consider the channel open.
-As the bitcoin in the channel is secured by the smart contract that manages that channel, payments settle instantly once received by the receiver and are not reversible.
+As the bitcoin in the channel is secured by the smart contract that manages that channel, payments settle instantly once received by the receiver.
+In practical terms, instant settlement means that payments take on the order of 1 second to execute.
```
Comment (Roasbeef) [inline ch03.asciidoc]: I'd say relax this to "on the order of a few seconds". EMV chips as is today can at times take up to 10 seconds to "complete" at physical PoS system, which people still perceive to be "instant".

**Review by Roasbeef — CHANGES_REQUESTED** [review]

File: ch03.asciidoc (hunk 1 above)
Comment (8go) [reply ch03.asciidoc]: Totally agree. We should not promise too much and always keep it realistic and factual.

Thanks for catching it.

Commit # 2 in c415d178ba8797d0f05396a87800dc234103e33d

File: ch03.asciidoc [hunk 2]
```diff
@@ -524,24 +524,37 @@ As payments are onion-routed, the users who route the payment will only be aware
...
 
-On the Lightning Network, a user can only send as much bitcoin as currently exists on their side of a particular channel.
+On the Lightning Network, a user can only send as much bitcoin as currently exists on their side of a particular channel to a channel partner.
 For instance, if a user owns one channel with 0.4 BTC on their side, and another channel with 0.2 BTC on their side, then the maximum they can send with one payment is 0.4 BTC.
 This is true regardless of how much bitcoin the user currently has in their Bitcoin wallet.
-At the time of writing, Atomic Multi-Path Payments (AMPs) are in development, which, in the above example, would allow the user to combine their 0.4 BTC and 0.2 BTC channels to be able to send a maximum of 0.6 BTC with one payment.
+
+At the time of writing, Atomic Multi-Path Payments (AMPs) are in development which, in the above example, would allow the user to combine both their 0.4 BTC and 0.2 BTC channels to be able to send a maximum of 0.6 BTC with one payment.
+
+In a direct payment where payer and payee are connected via a channel, the limit is states as above.
```
Comment (renepickhardt) [inline ch03.asciidoc]: stated?

File: ch03.asciidoc [hunk 3]
```diff
@@ -524,24 +524,37 @@ As payments are onion-routed, the users who route the payment will only be aware
...
+
+At the time of writing, Atomic Multi-Path Payments (AMPs) are in development which, in the above example, would allow the user to combine both their 0.4 BTC and 0.2 BTC channels to be able to send a maximum of 0.6 BTC with one payment.
+
+In a direct payment where payer and payee are connected via a channel, the limit is states as above. 
+If the payment gets routed, every routing node along the routing path must have at least the amount of funds as indicated in the payment. 
+If person A pays person C via person B, the payment follows the route A-B-C. 
+Say, person A wants to pay C the amount of 0.4 BTC. 
+Then both A as well as B must have 0.4 BTC available in their respective channels A-B and B-C.
+The same holds true for longer routes. 
+There must be enough capacity on every single channel that the payment is routed through.
```
Comment (renepickhardt) [inline ch03.asciidoc]: liquidity instead of capacity

**Review by renepickhardt — CHANGES_REQUESTED** [review]

**Pull request merged**